prow/kube/prowjob.go

```diff
@@ -79,22 +79,45 @@ type ProwJob struct {
 }
 
 type ProwJobSpec struct {
-	Type    ProwJobType  `json:"type,omitempty"`
-	Agent   ProwJobAgent `json:"agent,omitempty"`
-	Cluster string       `json:"cluster,omitempty"`
-	Job     string       `json:"job,omitempty"`
-	Refs    *Refs        `json:"refs,omitempty"`
-
+	Type      ProwJobType  `json:"type,omitempty"`
+	Agent     ProwJobAgent `json:"agent,omitempty"`
+	Cluster   string       `json:"cluster,omitempty"`
+	Job       string       `json:"job,omitempty"`
+	Refs      *Refs        `json:"refs,omitempty"`
+	ExtraRefs []*Refs      `json:"extra_refs,omitempty"`
```

Comment:
Would love to just have one slice here for cleanliness but we still need to declare one ref as the "main" ref under test

Reply:
I am fine with a separate slice for the extra repos.